src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Ensure that applying flex properties directly to the body element does not cause any unwanted side effects. It's uncommon to set `display: flex` on the `body` tag as it might lead to issues with overall page layout. Consider wrapping the content inside a container div if needed.